src/LowLevelMotionController.cpp

```diff
@@ -0,0 +1,34 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// LowLevelMotionController
+//
+// Class to implement control of the quads movement
+//
+////////////////////////////////////////////////////////////////////////////
+
+#include <ros/ros.h>
+#include "QuadVelocityController.hpp"
+
+using namespace Iarc7Motion;
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "Low_Level_Motion_Control");
+
+    ROS_INFO("Low_Level_Motion_Control begin");
+
+    ros::NodeHandle nh;
+
+    QuadVelocityController quadController(nh);
+
+    quadController.init();
+
+    while(ros::ok())
+    {
+        quadController.update();
```

Comment:
It depends on QuadVelocityController.cpp line 77 where it looks up the latest transform in the buffer. That blocks and I think it might yield, I'll have to look into it more.

If that does not yield then the velocity code will need a bit of a refactoring to detect changes in the transform. At that point I would say that should be a new feature that doesn't need fixing in initial structure.